eng/Version.Details.xml

```diff
@@ -26,9 +26,9 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>8470979eb44c2218025515234d3e01138bd74afb</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Web.Xdt" Version="3.0.0">
-      <Uri>https://github.com/aspnet/xdt</Uri>
-      <Sha>d4d088b6a9c793525b1a27a119cb66ba4587bb39</Sha>
+    <Dependency Name="Microsoft.Web.Xdt" Version="3.2.0">
```

Comment:
Is this absolutely necessary for the xliff work?

Basically, I'm thinking from a risk perspective, it might be better to separate these changes so that if we run into issues with the Microsoft.Web.Xdt version change it's easier to revert.